src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationRejection.scala

```diff
@@ -1,6 +1,22 @@
 package ch.epfl.bluebrain.nexus.admin.organizations
+
 import ch.epfl.bluebrain.nexus.commons.types.Rejection
 
 sealed trait OrganizationRejection extends Rejection
 
-case object OrganizationAlreadyExistsRejection extends OrganizationRejection
+/**
+  * Signals the the organization already exists.
+  */
+case object OrganizationAlreadyExists extends OrganizationRejection
+
+/**
+  * Signals that the organization does not exist.
+  */
+case object OrganizationDoesNotExist extends OrganizationRejection
+
+/**
+  * Signals that the provided revision does not match the latest revision
+  * @param latestRev    latest know revision
+  * @param providedRev  provided revision
+  */
+final case class IncorrectRevisionProvided(latestRev: Long, providedRev: Long) extends OrganizationRejection
```

Comment:
please rename to `IncorrectRev` to make it consistent with iam and rename fields to `latest` and `provided`